clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -103,6 +103,12 @@
         Utils.join(SecurityProtocol.names(), ", ") + ".";
     public static final String DEFAULT_SECURITY_PROTOCOL = "PLAINTEXT";
 
+    public static final String SOCKET_CONNECTIONS_SETUP_TIMEOUT_MS_CONFIG = "socket.connections.setup.timeout.ms";
```

Comment:
Didn't we decide to call this `socket.connection.setup.timeout.ms` during KIP discussion? (connection instead of connections)

Reply:
I think in the kip I used `socket.connections.setup.timeout.ms`
Since @cmccabe also suggests `socket.connection.setup.timeout.ms`, I'll change the KIP proposal to stick to `socket.connection.setup.timeout.ms`